packages/@aws-cdk/aws-apigateway/lib/integrations/lambda.ts

```diff
@@ -1,5 +1,6 @@
 import iam = require('@aws-cdk/aws-iam');
 import lambda = require('@aws-cdk/aws-lambda');
+import cdk = require('@aws-cdk/core');
```

Comment:
Import core classes directly (`import { Construct } from '@aws-cdk/core'`). The rational is that we think of core classes, like `Construct` as "part of the programming language".